src/pd/client.rs

```diff
@@ -210,10 +210,15 @@ impl PdClient for RpcClient {
         })?;
         check_resp_header(resp.get_header())?;
 
-        Ok(resp.take_store())
+        let store = resp.take_store();
+        if store.get_state() != metapb::StoreState::Tombstone {
+            Ok(store)
+        } else {
+            Err(Error::StoreTombstone(format!("{:?}", store)))
+        }
     }
 
-    fn get_all_stores(&self) -> Result<Vec<metapb::Store>> {
+    fn get_all_stores(&self, include_tombstone: bool) -> Result<Vec<metapb::Store>> {
```

Comment:
I think PD support such API is more elegant. BTW, how lightning interact with this API?

Reply:
Lightning doesn't interact with TiKV stores directly, Importer does (`src/import/import.rs` in this PR). 

Importer uses this API to get all TiKV nodes to forward the "switch to import mode" or "compact cluster" command. It doesn't affect SST ingestion.

Reply:
Lightning doesn't interact with TiKV stores directly, Importer does (`src/import/import.rs` in this PR). 

Importer uses this API to get all TiKV nodes to forward the "switch to import mode" or "compact cluster" command. It doesn't affect SST ingestion.

Reply:
Got it.

Reply:
Cc @nolouch What do you think? Is it ok to support this feature in PD side?

Reply:
I think it's ok to do that.